src/mmw/apps/home/urls.py

```diff
@@ -4,7 +4,13 @@
 from __future__ import division
 
 from django.conf.urls import patterns, url
-from apps.home.views import home_page, projects, project, project_clone
+from apps.home.views import (
+    home_page,
+    projects,
```

Comment:
Will re-order these to be alphabetical (`projects` should come after `project_`).